### ISSUE TITLE
[1.16.1] Error while disguising an armor stand via DisguiseAPI#disguiseToPlayers

### ISSUE THREAD

**Issue by Lix3nn53** [libraryaddict/LibsDisguises#489]
Error:
> java.lang.RuntimeException: Cannot modify the bounding box of a disguise which is not TargetType.SHOW_TO_EVERYONE_BUT_THESE_PLAYERS
	at me.libraryaddict.disguise.disguisetypes.Disguise.setModifyBoundingBox(Disguise.java:798) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.MobDisguise.setModifyBoundingBox(MobDisguise.java:149) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.MobDisguise.setModifyBoundingBox(MobDisguise.java:10) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.Disguise.clone(Disguise.java:218) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.TargetedDisguise.clone(TargetedDisguise.java:41) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.MobDisguise.clone(MobDisguise.java:70) ~[?:?]
	at me.libraryaddict.disguise.disguisetypes.MobDisguise.clone(MobDisguise.java:10) ~[?:?]
	at me.libraryaddict.disguise.DisguiseAPI.disguiseToPlayers(DisguiseAPI.java:312) ~[?:?]
	at me.libraryaddict.disguise.DisguiseAPI.disguiseToPlayers(DisguiseAPI.java:334) ~[?:?]
	at io.github.lix3nn53.guardiansofadelia.database.DatabaseManager.lambda$loadCharacterSelectionAndFormHolograms$3(DatabaseManager.java:156) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:400) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1059) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:354) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1007) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:846) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.1.jar:git-Spigot-758abbe-38de0b8]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_221]

Server version: 1.16.1 spigot
Plugin version: 10.0.12-SNAPSHOT (downloaded from discord)

Code:
`MobDisguise mobDisguise = new MobDisguise(DisguiseType.ARMOR_STAND, false);
LivingWatcher livingWatcher = mobDisguise.getWatcher();
livingWatcher.setInvisible(true);
livingWatcher.setNoGravity(true);
livingWatcher.setCustomNameVisible(true);

DisguiseAPI.disguiseToPlayers(armorStands.get(2), mobDisguise, player);
livingWatcher.setCustomName(ChatColor.GOLD + "Level: " + ChatColor.WHITE + level);
DisguiseAPI.disguiseToPlayers(armorStands.get(1), mobDisguise, player); //Line I get the error
livingWatcher.setCustomName(ChatColor.DARK_PURPLE + "Total Experience: " + ChatColor.WHITE + totalExp);
DisguiseAPI.disguiseToPlayers(armorStands.get(0), mobDisguise, player);
livingWatcher.setCustomName(ChatColor.GRAY + "Class: " + rpgClassCharacter.getClassString());`

Describtion:
armorStands is a List<ArmorStand> with 3 members. I am editing and using same MobDisguise in the code above so it seems like I need to create a new one for each armor stand. 

What I tried:
I created a new MobDisguise for each armor stand but another bug occured, no errors on console, only the last disguise is working in game. The other 2 armor stands' location changes like they are disguised but their CustomName does not change. This problem is not a 1.16 thing, I remember having `only the last disguise works` problem on older versions but I could not find a workaround this time.
Example, only armorStands.get(2) is disguised correctly:
`DisguiseAPI.disguiseToPlayers(armorStands.get(0), mobDisguiseThird, player);
DisguiseAPI.disguiseToPlayers(armorStands.get(1), mobDisguiseSecond, player);
DisguiseAPI.disguiseToPlayers(armorStands.get(2), mobDisguiseFirst, player);`

**Comment by libraryaddict** [comment]
Easy fix, if applying targeted disguise then set bounding box on it to not modify

**Comment by Lix3nn53** [comment]
Thanks for fast reply.

I tried `setModifyBoundingBox(false)` but still got the same error. I checked the Disguise class and Disguise#setModifyBoundingBox throws exception if `if (((TargetedDisguise)this).getDisguiseTarget() != TargetType.SHOW_TO_EVERYONE_BUT_THESE_PLAYERS) {` is true.
Since I am using `DisguiseAPI#disguiseToPlayers` TargetType is HIDE_DISGUISE_TO_EVERYONE_BUT_THESE_PLAYERS, right?

So I cant set ModifyBoundingBox to false? What should I do?
